src/dotnet/Fable.JS/Interfaces.fs

```diff
@@ -41,8 +41,9 @@ type IParseResults =
     abstract Errors: Error[]
 
 type IFableManager =
-    abstract CreateChecker: references:string[] * readAllBytes:(string->byte[])->IChecker
+    abstract CreateChecker: references:string[] * readAllBytes:(string->byte[]) -> IChecker
     abstract CreateCompiler: fableCorDir:string * ?replacements: seq<string * string> -> IFableCompiler
     abstract ParseFSharpProject: checker:IChecker * fileName:string * source:string->IParseResults
-    abstract GetCompletionsAtLocation: parseResults:IParseResults * line:int * col:int * lineText:string->Async<Completion[]>
-    abstract CompileToBabelJsonAst: com: IFableCompiler * parseResults:IParseResults * fileName:string * ?optimized:bool->string
\ No newline at end of file
+    abstract GetCompletionsAtLocation: parseResults:IParseResults * line:int * col:int * lineText:string -> Async<Completion[]>
+    abstract CompileToBabelAst: com: IFableCompiler * parseResults:IParseResults * fileName:string * optimized:bool -> Fable.AST.Babel.Program
+    abstract CompileToBabelJsonAst: com: IFableCompiler * parseResults:IParseResults * fileName:string * ?optimized:bool -> string
```

Comment:
Maybe make `optimized` optional for `CompileToBabelAst` too? We can also remove `CompileToBabelJsonAst`, when called from F# we can apply directly `JsInterop.toJson` and to call it from JS we probably need another layer to include Babel compilation anyways.

Reply:
@alfonsogarciacaro Alright, although explicit parameters force you to express the intent better than optional parameters.

Reply:
It's mainly for consistency, I'm also OK with the other way around: making `optimized` mandatory in `CompileToBabelJsonAst` (in case we keep it) :smile:

Reply:
Anyways, I'll merge this and we can do the amendment later :+1: